scripts/fred/homeownership/Homeownership_Rate.tmcf

```diff
@@ -0,0 +1,7 @@
+Node: E:FRED_Homeownership->E0
+typeOf: dcs:StatisticalVariable
+variableMeasured: dcs:Percent_Population_Homeowners
+measurementMethod: dcs:FRED
+observationAbout: C:COVIDTracking_States->GeoId
+observationDate: C:FRED_HomeownershipS->Date
+value: C:FRED_Homeownership->Percent_Population_Homeowners
```

Comment:
A couple comments to see if it helps with understanding:

The measurement method should be based on the original source (in this case, you can add it back with the particular Census Bureau survey that produced the data).

I think the `COVIDTracking_States` and `Fred_HomeownershipS` were typos rather than misconceptions.

Otherwise I think you got the hang of it-- just some misleading "Homeownership rate" that doesn't actually have "Owners" as first class objects but rather Housing Units. In our docs we may want to use this as an example of why people should always look at how a variable is defined by the source.

Reply:
Now, as promised: if we wanted to do CSV+TMCF without any CSV preprocessing, we could do this:


```
Node: E:MNHOWN->E0
typeOf: dcs:StatVarObservation
variableMeasured: dcid:Count_HousingUnit_OwnerOccupied_AsFractionOf_Count_HousingUnit_OccupiedHousingUnit
observationAbout: dcid:geoId/27
observationDate: C:MNHOWN->date
value: C:MNHOWN->value
```


Or:

```
Node: E:MNHOWN->E0
typeOf: schema:State
dcid: "geoId/27"

Node: E:MNHOWN->E1
typeOf: dcs:StatVarObservation
variableMeasured: dcid:Count_HousingUnit_OwnerOccupied_AsFractionOf_Count_HousingUnit_OccupiedHousingUnit
observationAbout: E:MNHOWN->E0
observationDate: C:MNHOWN->date
value: C:MNHOWN->value
```

And finally, preferably:

```
Node: E:MNHOWN->E0
typeOf: schema:State
geoId: "27"

Node: E:MNHOWN->E1
typeOf: dcs:StatVarObservation
variableMeasured: dcid:Count_HousingUnit_OwnerOccupied_AsFractionOf_Count_HousingUnit_OccupiedHousingUnit
observationAbout: E:MNHOWN->E0
observationDate: C:MNHOWN->date
value: C:MNHOWN->value

```

This last approach differs slightly from the 2nd in that users do not need to know the DCID of the place. This is helpful for non-US places, since we shouldn't expect users to know which standard (GeoID, WikidataID, NUTS ID, ISO codes, etc.) we use for that place's DCID.

Reply:
Thanks for these comments, @tjann ! This is really helpful. I've updated the tmcf file as per your suggestions.